lib/ansible/modules/windows/win_credential.ps1

```diff
@@ -513,19 +513,19 @@ Function ConvertTo-CredentialAttribute {
 }
 
 Function Get-DiffInfo {
-    param($Credential)
+    param($Ansible_Credential)
```

Comment:
If we really wanted to do this (and I don't think it is necessary), make it `AnsibleCredential` or `CredentialObj`.